recipes/p4api/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "p4api" %}
+{% set long_version = "2019.1.1796703" %}
+{% set version = "19.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ long_version }}"
+
+source:
+  url: http://ftp.perforce.com/perforce/r{{ version }}/bin.linux26x86_64/p4api.tgz   # [linux]
+  sha256: 3ef96f3df1bd93b492fc78c4441b7d59e3985709a5c065ea0756101cc0aa6a9c           # [linux]
+  url: http://ftp.perforce.com/perforce/r{{ version }}/bin.darwin90x86_64/p4api.tgz  # [osx]
+  sha256: 6d13930513352e621ef330e218ba552267f05a375209983ca497320c5a50909b           # [osx]
+  url: http://ftp.perforce.com/perforce/r{{ version }}/bin.mingw64x86/p4api.zip      # [win]
+  sha256: 342bfc820a988db11002ff1804f404796f06b214ba7630d6d8a3593aa564b7bf           # [win]
+
+build:
+  number: 0
+  script:
+    - cp -r include ${PREFIX}    # [unix]
+    - cp -r lib ${PREFIX}        # [unix]
+    - xcopy /E include %PREFIX%  # [win]
+    - xcopy /E lib %PREFIX%      # [win]
+  skip: True  # [win]
+
+requirements:
+  run:
+    - openssl
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/libclient.a            # [unix]
+    - test -f ${PREFIX}/lib/libp4script.a          # [unix]
+    - test -f ${PREFIX}/lib/libp4script_c.a        # [unix]
+    - test -f ${PREFIX}/lib/librpc.a               # [unix]
```

Comment:
Isn't this a separate project?

Reply:
Sorry, I didn't get you, could you please explain to me?

Reply:
Is librpc.a part of p4api?

Reply:
Yes, it is

Reply:
I was checking and you are right, actually ``librpc`` is not a separated packaged and p4 does not list it as a dependency.